docs/GettingStarted.md

```diff
@@ -75,26 +75,18 @@ console.log(ARC) // [class ARC]
 
 ## Setup - Python
 
-1. Install [python >3.11.x](https://www.python.org/downloads/)
-2. Create folder and put an `requirements.txt` file inside.
-3. Write `arctrl` and `fsspreadsheet` in two separate lines into it
-   ```
-   arctrl
-   fsspreadsheet
-   ```
-4. run `py -m pip install -r requirements.txt`
-
-   Of course you can replace the command `py` with anything that leads to the python executable of your liking.
+1. Install [python >=3.12](https://www.python.org/downloads/)
+2. Run `pip install arctrl`
```

Comment:
Actually, later I had to do `pip install fsspreadsheet` because python complained about not finding such a module. But I think it should in fact be possible to stick with a "one-step installation". Pip should be able to handle requirements without the user having to deal with it manually.

Reply:
Could a solution be to add "fsspreadsheet" to pyprojec.toml (in the root folder of this project) -> [build-system] -> requires ?